src/transformers/modeling_utils.py

```diff
@@ -2109,7 +2098,26 @@ def from_pretrained(cls, pretrained_model_name_or_path: Optional[Union[str, os.P
                     "Using `low_cpu_mem_usage=True` or a `device_map` requires Accelerate: `pip install accelerate`"
                 )
 
+        if load_in_8bit and bitsandbytes_config is not None:
+            raise ValueError(
+                "You can't pass both `load_in_8bit=True` and `bitsandbytes_config` as they are mutually exclusive."
+            )
```

Comment:
Yeah this makes me think supporting both bool and config to `load_in_8bit` is a better API.